docs/src/topics/degrees_of_freedom.md

```diff
@@ -40,9 +43,188 @@ dofs for the fields we added.
 close!(dh)
 ```
 
-## Ordering of Dofs
 
-!!! todo
-    Describe dof ordering within elements (vertices -> edges -> faces ->
-    volumes) and `dof_range`.
-    Describe (global) dof renumbering
+## Local DoF indices
+
+Locally on each element the DoFs are ordered by field, in the same order they are were added
```

Comment:
Should this really be documented or is this more akin to devdocs? When do you need to know this?

Reply:
You kinda need to know this for local assembly? Or do you mean that `dof_range` is enough?